Result of `nixpkgs-review pr 243572` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages failed to build:</summary>
  <ul>
    <li>gnunet-gtk</li>
    <li>taler-exchange</li>
    <li>taler-merchant</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>gnunet</li>
  </ul>
</details>

This breaks its dependents, those might require updates as well:

<details>

```
error: builder for '/nix/store/flnj2mddj7vpij69ja3lka11v4rdnwkd-gnunet-gtk-0.15.0.drv' failed with exit code 1;
       last 10 log lines:
       > checking for GLADE... yes
       > checking for gnutls... --with-gnutls not specified
       > checking gnutls/abstract.h usability... yes
       > checking gnutls/abstract.h presence... yes
       > checking for gnutls/abstract.h... yes
       > checking for gnutls_pubkey_import in -lgnutls... yes
       > checking for GNUnet core... /nix/store/ssq4z2zhlqp42ksx0n4gcbbmb74vk4v2-gnunet-0.19.4
       > checking for GNUnet util library in /nix/store/ssq4z2zhlqp42ksx0n4gcbbmb74vk4v2-gnunet-0.19.4... checking for gnunet/gnunet_util_lib.h... no
       > checking for gnunet/gnunet_conversation_service.h... no
       > configure: error: gnunet-gtk requires GNUnet
       For full logs, run 'nix log /nix/store/flnj2mddj7vpij69ja3lka11v4rdnwkd-gnunet-gtk-0.15.0.drv'.
error: builder for '/nix/store/i100a9i43cqr4flrrla0ad18vy46nh4f-taler-exchange-unstable-2022-07-17.drv' failed with exit code 2;
       last 10 log lines:
       >    49 | #include <gnunet/platform.h>
       >       |          ^~~~~~~~~~~~~~~~~~~
       > compilation terminated.
       > make[3]: *** [Makefile:1147: age_restriction.lo] Error 1
       > make[3]: Leaving directory '/build/exchange-93b45e6/src/util'
       > make[2]: *** [Makefile:524: all-recursive] Error 1
       > make[2]: Leaving directory '/build/exchange-93b45e6/src'
       > make[1]: *** [Makefile:508: all-recursive] Error 1
       > make[1]: Leaving directory '/build/exchange-93b45e6'
       > make: *** [Makefile:440: all] Error 2
       For full logs, run 'nix log /nix/store/i100a9i43cqr4flrrla0ad18vy46nh4f-taler-exchange-unstable-2022-07-17.drv'.
```

</details>

Note this is a duplicate of #198607 (which hasn't seen activity in a while but has the same build issues).

--- a/pkgs/applications/networking/p2p/gnunet/gtk.nix
+++ b/pkgs/applications/networking/p2p/gnunet/gtk.nix
@@ -13,11 +13,11 @@
 
 stdenv.mkDerivation rec {
   pname = "gnunet-gtk";
-  version = "0.15.0";
+  version = "0.19.0";
 
   src = fetchurl {
     url = "mirror://gnu/gnunet/${pname}-${version}.tar.gz";
-    sha256 = "sha256-FLLlqpQ7Bf+oNRUvx7IniVxFusy/tPYxEP2T6VGF7h8=";
+    sha256 = "sha256-MwAWs1rHXYlRUcAWX8LnCLTwEOSI68aA0s7uZGgYR3w=";
   };
 
   nativeBuildInputs= [

--- a/pkgs/applications/networking/taler/default.nix
+++ b/pkgs/applications/networking/taler/default.nix
@@ -4,23 +4,24 @@
 }:
 
 let
-  taler-merchant-backoffice = fetchgit {
-    url = "https://git.taler.net/merchant-backoffice.git";
-    # branch "prebuilt" as of 2022-07-01
-    rev = "1ef7150f32960cb65ebea67839cd5023f29a3d1d";
-    sha256 = "sha256-ZtLYWHi6l5DxFvDm8RFGUD0BiAfJXCZr/ggrP3Uw7/0=";
+  version = "0.9.2";
+
+  taler-wallet-core = fetchgit {
+    url = "https://git.taler.net/wallet-core.git";
+    rev = "v${version}";
+    sha256 = "sha256-DTnwj/pkowR1b1+N94pnuLykD2O37Nh8AKhUIzY7NaU=";
   };
 
 in rec {
   taler-exchange = stdenv.mkDerivation rec {
     pname = "taler-exchange";
-    version = "unstable-2022-07-17";
+    inherit version;
 
     src = fetchgit {
       url = "https://git.taler.net/exchange.git";
-      rev = "93b45e62eef254eae68bc119b9770e97bae2c9fa";
+      rev = "v${version}";
       fetchSubmodules = true;
-      sha256 = "sha256-BQxbwEf0wIkBOBVsPgMkMvUj4kFReXMUFTiSG0jXOJ0=";
+      sha256 = "sha256-c0cX38hDIZGVhHrD9LgDU70dF2AYuZmsakC8yDyZE54=";
     };
 
     nativeBuildInputs = [
@@ -42,7 +43,6 @@ in rec {
     ];
     propagatedBuildInputs = [ gnunet ];
 
-    configureFlags = [ "--with-gnunet=${gnunet}" ];
     preConfigure = ''
       ./contrib/gana-update.sh
     '';
@@ -73,15 +73,15 @@ in rec {
 
   taler-merchant = stdenv.mkDerivation rec {
     pname = "taler-merchant";
-    version = "unstable-2022-07-11";
+    inherit version;
 
     src = fetchgit {
       url = "https://git.taler.net/merchant.git";
-      rev = "960dcacf25e51cc2bff359ea1fc86cdd3d9e6083";
-      sha256 = "sha256-Wn11z6YjnylZl3z2JjBlrtZ1KHfQUHLIYWo5F+mAmNo=";
+      rev = "v${version}";
+      sha256 = "sha256-NPK8yhuTtZZiWE7OsUMdlb2aycegPzRFud41xHE9IL8=";
     };
     postUnpack = ''
-      ln -s ${taler-merchant-backoffice}/spa.html $sourceRoot/contrib/
+      ln -s ${taler-wallet-core}/spa.html $sourceRoot/contrib/
     '';
 
     nativeBuildInputs = [ pkg-config autoreconfHook ];

